utils/scarb-ui/src/verbosity.rs

```diff
@@ -14,6 +14,10 @@ pub enum Verbosity {
     ///
     /// String representation: `quiet`.
     Quiet,
+    /// Avoid printing warnings to standard output.
+    ///
+    /// String representation: `no-warnings`.
+    NoWarnings,
```

Comment:
This can be
```suggestion
    Error,
```
Indicating that it will print errors but not warnings

Reply:
That was my initial thought too, but we have some outputs that aren’t warnings or errors, like:
```rs
    ws.config()ui().print(Status::new("Packaging", &pkg_id.to_string()));
```
I figured we probably want to keep these messages and only mute warnings, so `NoWarnings` seemed more fitting.

Reply:
Yes, I would still use `ERROR` as it's more customary

Reply:
Are you sure it's more customary? Unlike `--quiet`, `--verbose` and `--no-warnings`, `--error` imo doesn't seem like a flag directly related to verbosity, and reading docs will be required to understand what it actually does.